### ISSUE TITLE
Can't move requests into folders (or folders into folders)

### ISSUE THREAD

**Issue by jovial-snyder** [getinsomnia/insomnia#6028]
### Expected Behavior

This issue is a continuation of the closed #5182.

When drag-and-dropping a request or a folder onto another folder, it should move that request into that folder.

### Actual Behavior

It does not.

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2023.2.2

### What operating system are you using?

Ubuntu

### Operating System Version

23.04

### Installation method

Snap

**Comment by ben-wallis** [comment]
I'm also having this same problem in on the 2023.2.2 Windows build.

**Comment by MMarquesGIT** [comment]
It's possible to move but it's very hard and time consuming.
My work around it's to create empty requests/folders and then drag the item between them.
You save a lot of time...

**Comment by abedwardsw** [comment]
confirm same issue for me, 
version: 2023.4.0
os: Darwin x64 22.5.0
arch: x64

**Comment by tsptylertech** [comment]
I have 2023.5.2 and I'm having this problem too...

**Comment by vermgit** [comment]
My Insomnia on Windows updated itself to 2023.5.3 and it reordered all folders and requests and I can't move them around anymore. The current order of them is completely useless.

Also all XPath-references are now broken. I don't know, maybe that's related somehow.

To me this bug is a blocker.

**Comment by jhinckley** [comment]
Same problem as @vermgit  on `2023.5.3`.  It's beyond frustrating.  

**update 1**  -- ~~I also can't import anything.  It sees everything in the file just fine but importing does absolutely BUPKIS.~~

**update 2**  --  Import works!  So this turned out to just be a viewing problem.  After you import on this version, nothing is visible and you can't seem to find anything but lo and behold, if you just click on that house/home icon, a fresh sidebar shows up, populated with all the imported items...multiplied by however many times you imported it while testing `*snicker snicker*`.   I think it would save us all a huge headache if the client would jump to that home screen right after import so we can immediately see that the content was actually imported.  It really is kind of sneaky how it appears empty right after a successful import.  

However, I still cannot move any requests to other folders, I can only move them out of a folder to the root.  This makes it really tough to organize requests and it seems like such a basic function that should have been fixed many versions ago.

**Comment by vermgit** [comment]
There was an update to 2023.5.4, which doesn't change anything. Same blocking behaviour.

**Comment by ehduardu** [comment]
I'm getting this error, all my collections are completely frozen

![bug_insomnia](https://github.com/Kong/insomnia/assets/32852890/9b2eaae9-b2ae-49d2-8b05-5d3adc589f64)

Version: `Insomnia 2023.5.4`
OS: `Ubuntu 22.04.3 LTS x64`

UPDATE: I reverted to `.deb` version `2023.4.0` and it works well

**Comment by vermgit** [comment]
There was an update to 2023.5.5, which doesn't change anything. Same blocking behaviour.

**Comment by tfedor** [comment]
Same issue, dragging requests into folders just doesn't work and it's beyond frustrating. If there at least was some other way how to set a folder for request...

**Comment by tsptylertech** [comment]
I have 2023.5.5 and am having the same issue.

**Comment by jhinckley** [comment]
I believe another poster already mentioned it but you can drag a request inbetween other requests and it will stay.  You have to collapse/expand the folder to show the changes.

**Comment by vermgit** [comment]
> I believe another poster already mentioned it but you can drag a request inbetween other requests and it will stay. You have to collapse/expand the folder to show the changes.

Not on my end. Requests remain where they are.

**Comment by jhinckley** [comment]
> > I believe another poster already mentioned it but you can drag a request inbetween other requests and it will stay. You have to collapse/expand the folder to show the changes.
> 
> Not on my end. Requests remain where they are.

It's kind of tricky because it has to be dropped perfectly in-between 2 other requests.  None of this even matters IMO because there's no way I'm going to use Insomnia in the long-term if they don't straighten out these monkeyshines.  

There are 530 open issues right now.  Something tells me this problem won't be high up on the list.

**Comment by mikebridge** [comment]
Same problems with reordered folders and and inability to move items on Linux:

```
Version: Insomnia 2023.5.5
Build date: 2023-08-18
OS: Linux x64 6.4.7-surface
Electron: 25.2.0
Node: 18.15.0
Node ABI: 116
V8: 11.4.183.23-electron.0
Architecture: x64
```

**Comment by vermgit** [comment]
2023.5.6 fixes this 😃  Thanks!

**Comment by jackkav** [comment]
fixed in #6323

**Comment by jhinckley** [comment]
The issue was fixed in the upgrade to 5.6 but it also removed all the color coding for requests.  Now they're just b&w text boxes. 

![image](https://github.com/Kong/insomnia/assets/2645132/a6537822-35d2-4976-afa2-7c79698cecb2)
